test/ProgressOnderwijsUtils.Tests/MetaObjectBulkCopyTest.cs

```diff
@@ -209,5 +235,32 @@ public void MetaObjectSupportsCustomObject_readonly()
             PAssert.That(() => fromDb.All(x => x.Bla3 == null));
         }
 
+#if NET461
+        [Fact]
+#else
+        [Fact(Skip = "MetaObjectBulkCopy does not have a way to set a transaction that's supported on .NET Core.")]
+#endif
+        public void MetaObjectSupportsCustomObject_struct()
+        {
+            PAssert.That(() => CustomBlaStruct.MethodWithIrrelevantName("aap").AsString == "aap");
+            CreateTempTable();
+            SampleObjects.BulkCopyToSqlServer(Context.Connection, "#MyTable");
+            var fromDb = SQL($"select Id, Bla, Bla2 from #MyTable order by Id").ReadMetaObjects<BlaOk_with_struct_property>(Context);

```

Comment:
Wat gebeurt er als een van deze `Bla2` waardes `null` is?  Gaat ie dan crashen, of `default(CustomBlaStruct)` doen?  Dat laatste lijkt me foutgevoelig.

Reply:
Test toegevoegd